tasks/deploy/config/mainnet/morpho-blue-wsteth-usda-182day.ts

```diff
@@ -0,0 +1,126 @@
+import { Address, encodeAbiParameters, parseEther, zeroAddress } from "viem";
+import {
+    HyperdriveInstanceConfig,
+    SIX_MONTHS,
+    USDA_ADDRESS_MAINNET,
+    getLinkerDetails,
+    normalizeFee,
+    parseDuration,
+    toBytes32,
+} from "../../lib";
+import { MAINNET_FACTORY_NAME } from "./factory";
+import { MAINNET_MORPHO_BLUE_COORDINATOR_NAME } from "./morpho-blue-coordinator";
+
+export const MAINNET_MORPHO_BLUE_WSTETH_USDA_182DAY_NAME =
+    "ElementDAO 182 Day Morpho Blue WSTETH/USDA Hyperdrive";
+
+const CONTRIBUTION = parseEther("100");
+
+const morphoBlueParameters = encodeAbiParameters(
+    [
+        {
+            components: [
+                {
+                    name: "morpho",
+                    type: "address",
+                },
+                {
+                    name: "collateralToken",
+                    type: "address",
+                },
+                {
+                    name: "oracle",
+                    type: "address",
+                },
+                {
+                    name: "irm",
+                    type: "address",
+                },
+                {
+                    name: "lltv",
+                    type: "uint256",
+                },
+            ],
+            name: "MorphoBlueParams",
+            type: "tuple",
+        },
+    ],
+    [
+        {
+            morpho: "0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb" as `0x${string}`,
+            collateralToken:
+                "0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0" as `0x${string}`, // Mainnet wstETH
+            oracle: "0xBC693693fDBB177Ad05ff38633110016BC043AC5" as `0x${string}`,
+            irm: "0x870aC11D48B15DB9a138Cf899d20F13F79Ba00BC" as `0x${string}`,
+            lltv: BigInt("860000000000000000"),
+        },
+    ],
+);
+
+export const MAINNET_MORPHO_BLUE_WSTETH_USDA_182DAY: HyperdriveInstanceConfig<"MorphoBlue"> =
+    {
+        name: MAINNET_MORPHO_BLUE_WSTETH_USDA_182DAY_NAME,
+        prefix: "MorphoBlue",
+        coordinatorAddress: async (hre) =>
+            hre.hyperdriveDeploy.deployments.byName(
+                MAINNET_MORPHO_BLUE_COORDINATOR_NAME,
+            ).address,
+        deploymentId: toBytes32(
+            MAINNET_MORPHO_BLUE_WSTETH_USDA_182DAY_NAME.slice(0, 32),
+        ),
+        salt: toBytes32("0x4201"),
+        extraData: morphoBlueParameters,
+        contribution: CONTRIBUTION,
+        fixedAPR: parseEther("0.035"),
+        timestretchAPR: parseEther("0.075"),
```

Comment:
With this timestretch we can expect that a 10k short with 1% slippage would require about `10,000*(33-0.7(14.99369775)/(0.04 + 1) =` 229,080.8805 in liquidity.

If we went with a 5% timestretch this would lower the liquidity requirement to:
`10000*(33-0.7*(A23)/(0.04 + 1))=` 178,621.3208 in liquidity

Reply:
Let's talk more about this next week, but this makes sense to me. This is a Morpho market, so it's conceivably possible that this could spike, but I've never seen this above 7.5%.